post_installation_configuration/preparing-for-users.adoc

```diff
@@ -100,6 +100,10 @@ include::modules/rbac-viewing-local-roles.adoc[leveloffset=+2]
 
 include::modules/rbac-adding-roles.adoc[leveloffset=+2]
 
+include::modules/unauthenticated-users-cluster-role-binding-con.adoc[leveloffset=+2]
+
+include::modules/unauthenticated-users-cluster-role-binding.adoc[leveloffset=+2]
```

Comment:
I would put this further down. I would put it after "**Creating a cluster admin**". And maybe prefix it with some introduction like:

It might be necessary to give access to unauthenticated users, you can do this by adding `system:unauthenticated` to a role binding with a reference to a (cluster) role, like we taught you in the chapter above.

And then we could give the module with the hint that the access has been removed and how to restore it.